.travis.yml

```diff
@@ -19,30 +19,79 @@ addons:
       - libgif-dev
       - google-chrome-stable
 
-install: # Install ChromeDriver (64bits; replace 64 with 32 for 32bits).
-  - wget -N https://chromedriver.storage.googleapis.com/78.0.3904.70/chromedriver_linux64.zip -P ~/
-  - unzip ~/chromedriver_linux64.zip -d ~/
-  - rm ~/chromedriver_linux64.zip
-  - sudo mv -f ~/chromedriver /usr/local/share/
```

Comment:
Could this not just become `sudo mv -f ~/chromedriver /usr/local/bin/` and then perform the `chmod -x`?
That should be in the `$PATH` env var right, means you can get rid of the symbolic link too.